### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by TheGreenBandit**

### CODE REVIEW

**Comment by TheGreenBandit** [comment]
Just removing old things from the readme

**Review by Yimura — APPROVED** [review]

**Comment by tupoy-ya** [comment]
Is there a reason to have ```git pull && git submodule update``` in the readme if it's already done in the script?

**Comment by TheGreenBandit** [comment]
Actually no, better remove that too i guess

File: README.md [hunk 1]
```diff
@@ -59,18 +59,14 @@ Now, you will be able to open the solution, and simply build it in Visual Studio
 ## Staying Up To Date
 
 Pull the latest changes from Github:
```
Comment (Yimura) [inline README.md]: Don't forget to remove this.

```
Pull the latest changes from Github:
```

**Review by Yimura — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (TheGreenBandit) [reply README.md:61]: Ah yes

Comment (tupoy-ya) [reply README.md]: > Don't forget to remove this.
> 
> ```
> Pull the latest changes from Github:
> ```

I think it shuld be:
```markdown
Pull the latest changes from Github and regenerate project files:
```
To not confuse people.

Comment (TheGreenBandit) [reply README.md:59]: Sure that could be done

Comment (TheGreenBandit) [reply README.md]: Wait I accidentally removed somethingg

**Review by Yimura — APPROVED** [review]

**Pull request merged**